mcs/class/System/System.Diagnostics/Process.cs

```diff
@@ -716,6 +716,16 @@ bool StartWithCreateProcess (ProcessStartInfo startInfo)
 
 				procInfo.envVariables = envVariables.ToArray ();
 			}
+			
+			if (startInfo.ArgumentList.Count > 0)
+			{
+				StringBuilder sb = new StringBuilder();
+				foreach (string argument in startInfo.ArgumentList)
+					PasteArguments.AppendArgument(sb, argument);
```

Comment:
Do we need to worry about quoting or anything here? I assume that is up to the client using `ArgumentList`?

Reply:
`PasteArguments.AppendArgument` does all that https://github.com/mono/corefx/blob/5b4edb9b0a721aed60c9c8ddeed441bdd7451e42/src/Common/src/System/PasteArguments.cs#L11